### ISSUE TITLE
[test_fdb_mac_move.py] | is failing on topology 'any', should be t0

### ISSUE THREAD

**Issue by slutati1536** [sonic-net/sonic-mgmt#7896]
**Description**

The fdb/test_fdb_mac_move.py::test_fdb_mac_move is passing only on t0 topology, but during the merge it was added

pytestmark = [

    pytest.mark.topology('any')

]
The issue is introduced by PR: [https://github.com/sonic-net/sonic-mgmt/pull/7789](https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsonic-net%2Fsonic-mgmt%2Fpull%2F7789&data=05%7C01%7Cslutati%40nvidia.com%7C5c7828dd3d884670096308db2ff7d411%7C43083d15727340c1b7db39efd9ccc17a%7C0%7C0%7C638156513697031014%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=6tACBcYGgu7H9%2F0iaCQemRscLRO3sTT1oBW14nilr00%3D&reserved=0)

**Steps to reproduce the issue:**
1. run fdb/test_fdb_mac_move.py::test_fdb_mac_move

**Describe the results you received:**

Test failing over:
    def test_fdb_mac_move(ptfadapter, duthosts, rand_one_dut_hostname, ptfhost, get_function_conpleteness_level):
        # Perform FDB clean up before each test
        fdb_cleanup(duthosts, rand_one_dut_hostname)
    
        normalized_level = get_function_conpleteness_level
        if normalized_level is None:
            normalized_level = "basic"
        loop_times = LOOP_TIMES_LEVEL_MAP[normalized_level]
    
        duthost = duthosts[rand_one_dut_hostname]
        conf_facts = duthost.config_facts(host=duthost.hostname, source="persistent")['ansible_facts']
    
        # reinitialize data plane due to above changes on PTF interfaces
        ptfadapter.reinit()
    
        router_mac = duthost.facts['router_mac']
    
        port_index_to_name = {v: k for k, v in list(conf_facts['port_index_map'].items())}
    
        # Only take interfaces that are in ptf topology
        ptf_ports_available_in_topo = ptfhost.host.options['variable_manager'].extra_vars.get("ifaces_map")
        available_ports_idx = []
        for idx, name in list(ptf_ports_available_in_topo.items()):
            if idx in port_index_to_name and conf_facts['PORT'][port_index_to_name[idx]].get('admin_status',
                                                                                             'down') == 'up':
                available_ports_idx.append(idx)
    
        vlan_table = {}
        interface_table = defaultdict(set)
        config_portchannels = conf_facts.get('PORTCHANNEL', {})
    
        # if DUT has more than one VLANs, use the first vlan
>       name = list(conf_facts['VLAN'].keys())[0]
E       KeyError: 'VLAN'

**Describe the results you expected:**
test should pass,
pytestmark = [
    pytest.mark.topology('any')
]
should be replaced by:
pytestmark = [
    pytest.mark.topology('t0')
]

**Additional information you deem important:**
**Output of show version:**

```
SONiC Software Version: SONiC.202211_RC8.3-696a6a43b_Internal
Distribution: Debian 11.6
Kernel: 5.10.0-18-2-amd64
Build commit: 696a6a43b
Build date: Mon Mar 27 10:50:51 UTC 2023
Built by: sw-r2d2-bot@r-build-sonic-ci03-244

Platform: x86_64-mlnx_msn4600c-r0
HwSKU: Mellanox-SN4600C-C64
ASIC: mellanox
ASIC Count: 1
Serial Number: MT2131X10295
Model Number: MSN4600-CS2FO
Hardware Revision: A1
Uptime: 13:16:06 up 13:25,  2 users,  load average: 1.37, 1.34, 1.30
Date: Tue 28 Mar 2023 13:16:06

Docker images:
REPOSITORY                                         TAG                               IMAGE ID       SIZE
docker-orchagent                                   202211_RC8.3-696a6a43b_Internal   84055ebb4c77   539MB
docker-orchagent                                   latest                            84055ebb4c77   539MB
docker-fpm-frr                                     202211_RC8.3-696a6a43b_Internal   83d5f57abddb   550MB
docker-fpm-frr                                     latest                            83d5f57abddb   550MB
docker-teamd                                       202211_RC8.3-696a6a43b_Internal   e6b58409a5c6   520MB
docker-teamd                                       latest                            e6b58409a5c6   520MB
docker-macsec                                      latest                            d76f633dda63   522MB
docker-syncd-mlnx                                  202211_RC8.3-696a6a43b_Internal   2bd090589266   939MB
docker-syncd-mlnx                                  latest                            2bd090589266   939MB
docker-platform-monitor                            202211_RC8.3-696a6a43b_Internal   4fcfc27b9bbe   942MB
docker-platform-monitor                            latest                            4fcfc27b9bbe   942MB
docker-sonic-telemetry                             202211_RC8.3-696a6a43b_Internal   be9f9e750657   802MB
docker-sonic-telemetry                             latest                            be9f9e750657   802MB
docker-snmp                                        202211_RC8.3-696a6a43b_Internal   62432cf85f40   549MB
docker-snmp                                        latest                            62432cf85f40   549MB
docker-eventd                                      202211_RC8.3-696a6a43b_Internal   855b7c3d7218   503MB
docker-eventd                                      latest                            855b7c3d7218   503MB
docker-dhcp-relay                                  latest                            c90d1c5967eb   513MB
docker-lldp                                        202211_RC8.3-696a6a43b_Internal   e33be345d3f2   546MB
docker-lldp                                        latest                            e33be345d3f2   546MB
docker-mux                                         202211_RC8.3-696a6a43b_Internal   e9f6fa7ef277   552MB
docker-mux                                         latest                            e9f6fa7ef277   552MB
docker-database                                    202211_RC8.3-696a6a43b_Internal   9cae58961a25   503MB
docker-database                                    latest                            9cae58961a25   503MB
docker-sonic-p4rt                                  202211_RC8.3-696a6a43b_Internal   77b20e5b758d   586MB
docker-sonic-p4rt                                  latest                            77b20e5b758d   586MB
docker-router-advertiser                           202211_RC8.3-696a6a43b_Internal   f3a99688e9df   503MB
docker-router-advertiser                           latest                            f3a99688e9df   503MB
docker-sflow                                       202211_RC8.3-696a6a43b_Internal   780973343beb   479MB
docker-sflow                                       latest                            780973343beb   479MB
docker-nat                                         202211_RC8.3-696a6a43b_Internal   83454bb6925c   481MB
docker-nat                                         latest                            83454bb6925c   481MB
docker-sonic-mgmt-framework                        202211_RC8.3-696a6a43b_Internal   20cfc12ab3c5   612MB
docker-sonic-mgmt-framework                        latest                            20cfc12ab3c5   612MB

```

**Comment by yxieca** [comment]
@yutongzhang-microsoft can you check why the testcases was in 'any' category?

**Comment by wangxin** [comment]
Fixed in #7963
